### DISCUSSION TITLE
Multi File Upload using attach

### DISCUSSION THREAD

**Discussion by amitInnovo** [getgauge/taiko#2450] [General]
Using attach as part of Taiko (v1.2.8), while attempting to pass an array of file paths as the first argument, I get the following error message:
Argument of type 'string[]' is not assignable to parameter of type 'string'.ts(2345)

Is there something I am doing wrong? Using VS Code.

**Comment by zabil** [comment]
Please try passing individual files instead on an array of files names. You can call the API multiple times. I do not thing an array of files is supported at the moment.

**Reply by amitInnovo** [reply]
Ok thanks. I was asking based on [#1859]([https://github.com/getgauge/taiko/discussions/1859](url) ) comment that the feature was introduced in 1.2.2.
